apps/prairielearn/src/lib/authz-data-lib.ts

```diff
@@ -122,9 +126,7 @@ export const CourseOrInstanceContextDataSchema = z.object({
 });
 export type CourseOrInstanceContextData = z.infer<typeof CourseOrInstanceContextDataSchema>;
 
-export type AuthzDataWithoutEffectiveUser =
-  | ConstructedCourseOrInstanceSuccessContext['authzData']
-  | DangerousSystemAuthzData;
+export type AuthzDataWithoutEffectiveUser = PlainAuthzData | DangerousSystemAuthzData;
```

Comment:
IMO this is an improvement; it has better symmetry with `AuthzDataWithEffectiveUser` now.